xui/additional_files_unzipped/s3_browser/s3_browser/aws_bucket_browser_operations/aws_bucket_browser_operations.py

```diff
@@ -261,15 +261,15 @@ def is_versioning_enabled(s3_client, resource_id, default_region):
         return False
 
     except (CustomField.DoesNotExist, CustomFieldValue.DoesNotExist) as error:
-        # This error can arise when s3 buckets are added from discovery. In which case, attempt to find versioning a last with using the default region
-        logger.error(f"CustomField 's3_bucket_region' does not exist. Trying default_region={default_region}")
+        # This error can arise when s3 buckets are added from discovery and their s3_bucket_region parmeter is not set. As a backup, attempt to find versioning info using the default region
+        logger.warning(f"CustomField 's3_bucket_region' does not exist. Trying default_region={default_region}")
         s3_bucket = aws.get_boto3_resource(region_name=default_region, service_name="s3")
```

Comment:
will this raise another exception if the bucket isn't in the default region? if so, the code should probably catch that and return False (assuming the rest of the functionality works when we can't figure out the region)

Reply:
So learned something interesting testing this with random regions.  Changing the `default_region` to `us-west-1` didn't error, and correctly found the versioning status for the bucket I was looking at (`ui-demo-experience` in `us-east-1` 

Not only that by using that same connection I could look up info like `demo-shantam` which is in `ap-south-1`

(I should add that my RH for AWS only has `us-east-1` and `us-west-2` included
...which means you don't have to initialize the connection with the specific region that the resource is in right now, just a region that is supported by the AWS connection
![Screenshot 2023-12-18 at 2 10 51 PM](https://github.com/CloudBoltSoftware/applet-s3-bucket-browser/assets/105660893/9d2718e9-a91a-45b6-98f1-4bac610deb0d)
So using that theory i set the default_region to `ap-east-1` and then got this 500 error: 

```
botocore.exceptions.ClientError: An error occurred (IllegalLocationConstraintException) when calling the GetBucketVersioning operation: The ap-south-1 location constraint is incompatible for the region specific endpoint this request was sent to.
```
So I think we don't need another error message for now because most errors after it tries a valid default_region are going to be botocore errors, I think

Reply:
Nice exploration! So does using us-east-1 work for all buckets everywhere? If so, can we just remove all the logic about trying to read custom fields and figure out which region to use, and just always use us-east-1?

Reply:
Not if their account isn't enabled for 'us-east-1', which presumably is a possibility. It still makes sense to me to try and use what should be a valid user-specified input to attempt this connection

Reply:
ok, cool. I'm good with it.